website/content/02-guides/09-recipes.mdx

```diff
@@ -42,44 +40,44 @@ title: Recipes
    yarn prisma migrate save --experimental && yarn prisma migrate up --experimental
    ```
 
-1. Now your app has the following
-
-   1. `nexus-prisma` Nexus plugin allowing e.g.:
-
-      ```diff
-      +++ src/graphql.ts
-        objectType({
-          name: 'User',
-          definition(t) {
-      -     t.id('id)
-      -     t.string('name')
-      +     t.model.id()
-      +     t.model.name()
-          },
-        })
-      ```
-
-   2. An instance of the generated Prisma Client is a added to context under the `db` property, allowing e.g.:
-
-      ```diff
-      +++ src/graphql.ts
-        queryType({
-          definition(t) {
-            t.list.field('users', {
-              type: 'User',
-      -       resolve() {
-      -         return [{ id: '1643', name: 'newton' }]
-      +       resolve(_root, _args, ctx) {
-      +         return ctx.db.users.findMany()
-              },
-            })
-          },
-        })
-      ```
-
-   3. The TypeScript types representing your Prisma models are registered as a Nexus data source. In short this enables proper typing of `parent` parameters in your resolves. They reflect the data of the correspondingly named Prisma model.
-
-## Create a Consumable Plugin
+Your app now includes the following:
+
+`nexus-prisma` Nexus plugin allowing e.g.:
+
+    ```diff
```

Comment:
Does this have to be `ts` maybe? (and all others in this file)

Reply:
Up for discussion, I guess the goal was a chap way to show the before/after effect of what using this plugin looks likes.

Recipes section is one of our least touched areas so no strong opinion right now.

Reply:
You can use `ts` here - the +/- lines will still be highlighted - have confirmed that we can retire `diff`: https://github.com/prisma/docs/issues/526

I'll add a GH issue to swap out all `diff` blocks.